build.zig

```diff
@@ -24,12 +24,18 @@ pub fn build(b: *std.build.Builder) void {
         options.addOption(?[]const u8, "git_commit", null);
     }
 
+    options.addOption(
+        ConfigBase,
+        "config_base",
+        b.option(ConfigBase, "config", "Base configuration.") orelse .default,
+    );
+
     const tracer_backend = b.option(
-        TracerBackend,
+        config.TracerBackend,
```

Comment:
Now that TracerBackend is in config, we can use it in https://github.com/tigerbeetledb/tigerbeetle/blob/dec1160d09494d9853a166153b953b9f6d6cdc59//src/clients/node/src/node.zig#L32 too.